validate_vrt.py

```diff
@@ -2,6 +2,7 @@
 import json
 import jsonschema
 import sys
+# Don't create *.pyc files
 sys.dont_write_bytecode = True
```

Comment:
Are we removing this?  I don't recall if it had something to do with the build process or if its just better solved by the gitignore.

Reply:
Oh right. Gone now.